### PR TITLE
LRQA-71733, LRQA-50743 | master 

### PR DESCRIPTION
**Pull request by epark18**
https://issues.liferay.com/browse/LRQA-71733
https://issues.liferay.com/browse/LRQA-50743

### CODE REVIEW

**Comment by epark18** [comment]
ci:test:sf

File: portal-web/test/functional/com/liferay/portalweb/paths/portlet/gdpr/ExportPersonalData.path [hunk 1]
```diff
@@ -35,6 +35,11 @@
 	<td>//table//td[contains(.,'In Progress')]</td>
 	<td></td>
 </tr>
+<tr>
+	<td>EXPORT_PROCESS_MESSAGE</td>
```
Comment (patriciajeanperez) [inline portal-web/test/functional/com/liferay/portalweb/paths/portlet/gdpr/ExportPersonalData.path:39]: @epark18 have we tried a generic locator such as Message#EMPTY_INFO?

**Review by patriciajeanperez — COMMENTED** [review]
Good job on the refactor! @epark18 I just needed confirmation if the variables were set correctly in deleteAndAnonymizeAllEntriesCP and if we could use a generic path for the empty info message in the export process rather than creating a new one. Thank you!

File: portal-web/test/functional/com/liferay/portalweb/macros/GDPR.macro [hunk 2]
```diff
@@ -160,7 +161,8 @@ definition {
 	macro deleteAndAnonymizeAllEntriesCP {
 		GDPR.filterByApplications(
 			applications = "All Applications",
-			itemsCount = "${itemsCount}");
+			applicationItemsCount = "${itemsCount}",
+			totalItemsCount = "${itemsCount}");
```
[the remark this answers is not in the document]
Comment (epark18) [reply portal-web/test/functional/com/liferay/portalweb/macros/GDPR.macro:165]: oh, you're right! I missed that, my bad I will make the change.

File: portal-web/test/functional/com/liferay/portalweb/paths/portlet/gdpr/ExportPersonalData.path (hunk 1 above)
Comment (epark18) [reply portal-web/test/functional/com/liferay/portalweb/paths/portlet/gdpr/ExportPersonalData.path:39]: That probably would work, I will resend the changes :] Thanks Patricia!

**Comment by epark18** [comment]
ci:test:sf

**Comment by epark18** [comment]
ci:test:sf

**Comment by epark18** [comment]
ci:tsest:sf

**Comment by epark18** [comment]
ci:test:sf

**Comment by epark18** [comment]
ci:test:sf

**Comment by patriciajeanperez** [comment]
ci:forward

**Pull request closed without merging**